### ISSUE TITLE
RFE: Improve and separate membership process for Keptn Core and Keptn Integrations

### ISSUE THREAD

**Issue by christian-kreuzberger-dtx** [keptn/community#90]
### Pain

The current [Membership process](https://github.com/keptn/community/blob/master/COMMUNITY_MEMBERSHIP.md) is heavily focused on Keptn Core (github.com/keptn), but also mentions keptn-contrib and keptn-sandbox, e.g.:
![image](https://user-images.githubusercontent.com/56065213/147553997-71f219d4-5e39-4377-8593-04e244c9d1ed.png)

### Proposal

I propose to separate the process, such that we have
* an extensive process, as currently given in [Membership process](https://github.com/keptn/community/blob/master/COMMUNITY_MEMBERSHIP.md) for
  * Keptn Core / github.com/keptn
  * keptn-contrib - here the purpose is that some people need to have super-powers (not necessarily  be the same people as for keptn core); it should be possible for people that contribute to multiple repos in keptn-contrib to become somebody with super-powers in keptn-contrib
* a moderate process for github.com/keptn-contrib **per repository** (e.g., how to become a contributor of prometheus-service)
* Some sort of process to become a manager for keptn-sandbox (e.g., accept new repositories/contributions) - please note that repositories in keptn-sandbox are always owned by an individual or by a group of people, and organization maintainers should only interfere in consultation with the repository owners

In short, I think that we need to have some groundrules:
* Being a keptn core maintainer does **NOT NECESSARILY** make you a keptn-contrib maintainer (and vice versa)
* Being a **keptn-contrib** organization maintainer could be a very organizational role that is hard to get to (e.g., only after multiple contributions and being a long-term member)
* Being an organizational maintainer does not mean that you need to approve PRs, contribute code, etc... - this should be left to the repository owners (e.g., in keptn-contrib)

### Sponsors

**Please add yourself if you want to sponsor this**

### Checklist

- [ ] The idea is brought up for a discussion in a relevant Keptn Slack channel (use `#keptn-project` by default)
- [ ] Each sponsor should reply to this issue with the comment "*I support*".

**Comment by oleg-nenashev** [comment]
It is a counterpart of #84 , IMHO need to be done in parallel

**Comment by christian-kreuzberger-dtx** [comment]
> It is a counterpart of #84 , IMHO need to be done in parallel

Yes, I think both ladders (code and non-code) could/should include "organization manager". I don't think it's worth splitting it at the moment.

**Comment by thschue** [comment]
This goes in the same direction as #240. Therefore, closing this issue.
